dynawo/sources/Modeler/Common/DYNModelMulti.h

```diff
@@ -544,6 +544,34 @@ class ModelMulti : public Model, private boost::noncopyable {
    */
   void cleanBuffers();
 
+  /**
+   * @brief Process Y connectors registered as connecting only external variables
+   *
+   * Since we cannot connect external variables by buffer if there is no reference variable, we'll create a fictive
+   * variable for each external variable and connect these fictionous variables using Y connectors.
```

Comment:
typo : fictionous -> fictitious

Reply:
corrected to "fictive" to be compliant with the class name